src/sentry/integrations/slack/views/link_team.py

```diff
@@ -202,17 +240,23 @@ def handle(self, request: Request, signed_params: str) -> HttpResponseBase:
                 team_id=team.id,
                 types=[NotificationSettingEnum.ISSUE_ALERTS],
             )
+
         message = SUCCESS_LINKED_MESSAGE.format(
             slug=team.slug,
             workflow_addon=" and workflow" if has_team_workflow else "",
             channel_name=channel_name,
         )
+
         integration_service.send_message(
             integration_id=integration.id,
             organization_id=team.organization_id,
             channel=channel_id,
             message=message,
         )
+
+        _logger.info("team.linked_success", extra=logger_params)
```

Comment:
Do we need both log as well as metric?

Reply:
your right, ill remove it from here and a few other places i added this log.